backend/app/views/spree/admin/products/_autocomplete.js.erb

```diff
@@ -1,19 +1,25 @@
 <script type='text/template' id='product_template'>
-  <li id='product_{{ product.id }}' data-product-id='{{product.id }}' class='well small_product'>
-    <table class='table table-condensed table-bordered no-marginb'>
-      <tr>
-        <td class='image'>
-          {{#if product.image }}
-            <img src='{{product.image}}' alt='{{ product.name }}' class='thumbnail' />
-          {{ else }}
-            <img src='<%= image_path("noimage/small.png") %>' alt='{{product.name }}' class='thumbnail' />
-          {{/if}}
-        </td>
-        <td>
-          <a class="name">{{ product.name }}</a><br>
-          <span class="price">{{ product.display_price }}</span>
-        </td>
-      </tr>
-    </table>
+  <li id='product_{{ product.id }}' data-product-id='{{ product.id }}' data-taxons="{{ product.taxon_ids }}" class='product'>
+    <a class="sort-handle js-sort-handle"><span class="icon icon-move"></span></a>
+    <div class="btn-group">
+      <button type="button" class="btn btn-primary btn-sm dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu" role="menu" data-hook="taxon_product_dropdown">
+        <li><a href="/admin/products/{{ product.slug }}/edit"><%= Spree.t(:edit_product) %></a></li>
```

Comment:
This needs to use url helpers otherwise it won't work when people mount their spree store to somewhere other than root route.